static/esp_auto.css

```diff
@@ -231,7 +231,7 @@ table.multirow th,table.normal th {
 	background-color:#444;
 	color:#fff;
 	padding:6px;
-	width:100vw 
+	width: 100vw
```

Comment:
Does this space actually do something in a CSS?

Reply:
it keeps you busy commenting 🙂

Reply:
... and it improves readability, spaces and newlines will be removed by the minifier, so no need to do that in advance if it confuzzles with readability.

Reply:
true... the missing spaces came from one of the css beautifiers or validators... next time i do a format